apps/bluetooth-music/app.js

```diff
@@ -28,27 +29,30 @@ function resetLastUrl () {
 }
 
 function onAudioFocusGained () {
-  logger.debug('onAudioFocusGained')
+  logger.info('++++ onAudioFocusGained')
+  hasFocus = true
   a2dp.unmute()
 }
 
 function onAudioFocusLost (transient, mayDuck) {
-  logger.debug('onAudioFocusLost, transient =', transient)
+  logger.info('---- onAudioFocusLost, transient =', transient)
```

Comment:
Ditto.

Reply:
And recommendation is to use string template here, too :)

Reply:
removed